docs/index.md

```diff
@@ -1,42 +1,13 @@
-Installation Guides
-===================
+Open Science Grid Site Documentation
+====================================
```

Comment:
Use the # method for headings, IMO.

Reply:
I prefer the `====` and `----` method for h1 vs h2 headers, respectively. It's easy to spot and search for them in the source.